lib/endpoint/tags/table.js

```diff
@@ -112,6 +112,10 @@ function rowToParameter (i, el) {
     required: isRequired
   }
 
+  if (params.name === 'comments') {
+    params.type = 'objects[]'
+  }
```

Comment:
I was thinking of updating this to be:

```
// 'array ... objects' => 'objects[]'
if (params.type.match(/array.*?objects/)) {
  params.type = 'objects[]'
}
```

and it would fix https://github.com/octokit/routes/issues/60 as well

Reply:
yes I like that! How about a slight variation:

```js
if (/array of.*objects/.test(params.type)) {
  params.type = 'objects[]'
}
```

I think it might be slightly more readable?

Reply:
when you make the change, make sure to add a `fixes #60` to the pull request description :)

Reply:
sounds great, I'll make the change